api/index.js

```diff
@@ -13,26 +13,30 @@ const upload = multer({ storage });
 
 /// CONTACTS ROUTES
 
-router.get("/contacts", auth, ctrlContacts.getAll);
-router.get("/contacts/:id", auth, ctrlContacts.getById);
-router.post("/contacts", auth, ctrlContacts.addContact);
-router.delete("/contacts/:id", auth, ctrlContacts.removeContactById);
-router.put("/contacts/:id", auth, ctrlContacts.updateContact);
-router.patch("/contacts/:id/favorite", auth, ctrlContacts.updateStatus);
+router
+  .get("/contacts", auth, ctrlContacts.getAll)
+  .get("/contacts/:id", auth, ctrlContacts.getById)
+  .post("/contacts", auth, ctrlContacts.addContact)
+  .delete("/contacts/:id", auth, ctrlContacts.removeContactById)
+  .put("/contacts/:id", auth, ctrlContacts.updateContact)
+  .patch("/contacts/:id/favorite", auth, ctrlContacts.updateStatus);
```

Comment:
Sweet :-)

Reply:
Dziękuję, miałem nadzieję że to zobaczysz i się spodoba :)

Reply:
🤣🤣🤣

😈😈😈🤘🤘🤘